src/routes.js

```diff
@@ -500,14 +665,21 @@ const routes = [
     path: '/dl/*',
     component: 'Download',
     loading: 'page',
-    key: () => '/dl'
+    key: () => '/dl',
+    meta: {
+      title: { parts: () => [i18n.t('title.download')] }
```

Comment:
I actually don't think I see a `title.download` message.

Reply:
Whoops, added it back! This is one route i'm not sure how to test.